rules/no-inheritance.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @fileoverview Discourage use of inheritance
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+module.exports = {
+
+    meta: {
+
+        docs: {
+            recommended: true,
+            type: 'error',
+            description: 'Discourage use of inheritance',
+        },
+
+        schema: []
+
+    },
+
+    create: function (context) {
+
+        function inspectContractStatement(emitted) {
+            var node = emitted.node;
+            if (!emitted.exit) {
+                if (node.is.length > 0) {
+                    context.report({
+                        node: node,
+                        message: 'Avoid using inheritance'
```

Comment:
Please change message to `Avoid using inheritance for Contract <CONTRACT NAME>`.
The contract name can be accessed from the `node` object.